.github/config/cr.yaml

```diff
@@ -0,0 +1,6 @@
+# Set this to true for GPG signing
+sign: false
+# When `sign` is set to true, Add UID here and Add a step in release workflow to prepare and sign the GPG key.
```

Comment:
have removed the GPG steps. Added a reference here incase we want to do it in future